tinyagentos/chat/message_store.py

```
@@ -80,6 +81,11 @@ async def init(self) -> None:
         except Exception:
             # column already exists (SQLite raises on duplicate column name)
             pass
+        try:
+            await self._db.execute("ALTER TABLE chat_messages ADD COLUMN expires_at REAL")
+            await self._db.commit()
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)